test/functional/http/get/node/node.js

```diff
@@ -57,7 +57,7 @@ describe('GET /node', () => {
 			return expect(constantsResponse)
 				.to.have.property('version')
 				.to.match(
-					/^([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})(-(alpha|beta|rc)\.[0-9]{1,3})?$/
+					/^([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})(-(alpha|beta|rc)\.[0-9]{1,3}(\.[0-9]{1,3})?)?$/
```

Comment:
What is that change for?

Reply:
The test was failing due as we were using wrong regular expression. Was not identified before as in test config there was `0.0.0` version always.